config/locales/en.yml

```diff
@@ -219,8 +219,8 @@ en:
     job_count_plural_without_search: 'There are %{count} jobs listed'
     job_count: '%{count} job matches your search'
     job_count_plural: '%{count} jobs match your search'
-    job_count_with_location_category: '%{count} teaching job in %{location}'
-    job_count_plural_with_location_category: '%{count} teaching jobs in %{location}'
+    job_count_with_location_category: '%{count} job match near %{location}'
```

Comment:
I'd double check this one with content

Reply:
its the copy taken from chris's figma doc